global-script.lua

```diff
@@ -2049,6 +2049,18 @@ posMap = { -- This order should exactly match alternateSetupNames table
             Vector(-4.22, 1.08, 18.91),
             Vector(13.78, 1.08, 19.09),
         },
+        { -- Fragment 2
```

Comment:
instead of Fragment 2, I think I Inverted Fragment is a better/more interesting name (which came from the bgg link i sent you)